pkg/process/metadata/workloadmeta/workloadmeta.go

```diff
@@ -29,18 +29,29 @@ type ProcessEntity struct {
 type WorkloadMetaExtractor struct {
 	// Cache is a map from process hash to the workloadmeta entity
 	// The cache key takes the form of `pid:<pid>|createTime:<createTime>`. See hashProcess
-	cache      map[string]*ProcessEntity
-	cacheMutex sync.RWMutex
+	cache        map[string]*ProcessEntity
+	cacheVersion int32
```

Comment:
any reason not to use a `uint32` for the cacheVersion?

Reply:
I used `int32` to avoid an additional conversion to use it as `EventID` which is defined as `int32`:
https://github.com/DataDog/datadog-agent/blob/b7c1e7b582813e2e9562294066dda60388913461/pkg/proto/datadog/process/workloadmeta_process.proto#L8